invenio_app_rdm/administration/records/records.py

```diff
@@ -26,6 +26,7 @@ class RecordAdminListView(AdminResourceListView):
     resource_config = "records_resource"
     search_request_headers = {"Accept": "application/vnd.inveniordm.v1+json"}
     title = "Records"
+    menu_label = "Records"
```

Comment:
Minor: Could you please mark the introduced `menu_label` for translation?
```suggestion
    menu_label = _("Records")
```